src/test/run-pass/mir_trans_calls.rs

```diff
@@ -88,6 +88,19 @@ fn test8() -> isize {
     Two::two()
 }
 
+#[link(name = "c")]
+extern {
+    fn printf(format: *const u8, ...) -> i32;
```

Comment:
Other functions are in `fcntl`, `ioctl`, `execl{,e,p}`. Exec might be the best one I think.